kolibri_explore_plugin/settings.py

```diff
@@ -0,0 +1,6 @@
+from kolibri.deployment.default.settings.base import *  # noqa @UnusedWildImport
+
+
+MIDDLEWARE = list(MIDDLEWARE) + [  # noqa F405
+    "kolibri_explore_plugin.middleware.AlwaysAuthenticatedMiddleware"
+]
```

Comment:
Instead of adding this settings file here, do you think we could add it to kolibri-app _itself_ in endlessm/kolibri-app#30? For example, that repo already has a `django_app_settings.py` which is used for the Android app.